specification.md

```diff
@@ -0,0 +1,54 @@
+# RFC Websocket Specification Proposal - 0.0.1
+
+> A websocket only specification.
+
+# TOC
+- [General](#general)
+- [Authentication](#Authentication)
+- [Event base type](#event-base-type)
+- [API](./api/)
+    - [Party API](./api/party-api.md)
+    - [Game API](./api/game-api.md)
+    - [Chat API](./api/chat-api.md)
+    - [Errors](./errors.md)
+- [Workflows](./usecases/wip.md)
+- [TODOs](#todos)
+
+
+# General
```

Comment:
Message IDs:

- they should be Uuidv4 or similar
- it might be worth mentioning why they are required here:
  1) Player plays cards and frontend sents the event and expects a response
  2) Frontend gets a Message event instead, which it has to handle
  3) Frontend receives the response for the Play event.

  Both frontend and backend need to handle these cases. I didn't notice that when first reading the PR, some others might not aswell.

Reply:
Short info added. Might need another example... we dont have a response for the player plays card event. This would be the GameStateEvent which is not bound to a specific request and should be able to be received and processed at any point in time during a given game ...